donate/settings/base.py

```diff
@@ -0,0 +1,385 @@
+import django
+import logging.config
+
+from .environment import (
+    env,
+    app,
+    root
+)
+
+
+class Base(object):
+    SECRET_KEY = 'CHANGE ME OUTSIDE OF DEV'
+    WAGTAIL_SITE_NAME = 'donate'
+    WSGI_APPLICATION = 'donate.wsgi.application'
+    RECAPTCHA_ENABLED = False
+    LANGUAGE_CODE = 'en-US'
+    TIME_ZONE = 'UTC'
+    USE_I18N = True
+    USE_L10N = True
+    USE_TZ = True
+    AUTH_USER_MODEL = 'users.User'
+    DJANGO_LOG_LEVEL = env('DJANGO_LOG_LEVEL')
+    ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+    RANDOM_SEED = env('RANDOM_SEED')
+
+    # Domain Redirects
+    DOMAIN_REDIRECT_MIDDLEWARE_ENABLED = env('DOMAIN_REDIRECT_MIDDLEWARE_ENABLED')
+    TARGET_DOMAINS = env('TARGET_DOMAINS')
+
+    # Initialize these attributes to None
+    BASKET_API_ROOT_URL = env('BASKET_API_ROOT_URL')
+    BASKET_SQS_QUEUE_URL = env('BASKET_SQS_QUEUE_URL')
+
+    # Pontoon settings
+    WAGTAILLOCALIZE_PONTOON_SYNC_MANAGER_CLASS = 'donate.core.pontoon.CustomSyncManager'
+    WAGTAILLOCALIZE_PONTOON_GIT_URL = env('WAGTAILLOCALIZE_PONTOON_GIT_URL')
+    WAGTAILLOCALIZE_PONTOON_GIT_CLONE_DIR = env('WAGTAILLOCALIZE_PONTOON_GIT_CLONE_DIR')
+
+    # Static content settings
+    STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+    STATICFILES_DIRS = [app('frontend')]
+    STATIC_ROOT = root('static')
+    STATIC_URL = '/static/'
+
+    MEDIA_ROOT = root('media/')
+    MEDIA_URL = '/media/'
+
+    HEROKU_APP_NAME = ''
```

Comment:
This feels like it wants to be an env var? (with `'''` fallback?)

Reply:
This is really only ever needed when using the `ReviewApp` configuration, so I can probably drop it from base.py altogether.

Reply:
yes